nix/devShell.nix

```diff
@@ -43,6 +43,7 @@
 
                 # for some build scripts
                 jq
+                gnused # not installed by default on MacOS!
```

Comment:
this is required for the fixup.sh sed script to work on MacOS, otherwise it'll use the system sed which will break terribly